hedyorg/hedy
# 🪲 Add support for numbers in return statements

Fixes #5170
With this PR, `return` statements try to convert their output to integers and floats and output strings if the number conversion fails. This change is required because the output of a function cannot be used in an arithmetic operation.

**How to test**
Navigate to level 14 and run the following program. Ensure that it completes without an error:
```
define round with x
    return x

r = call round with 1
print r + 1
```
